$OpenBSD: patch-lib_talloc_wscript,v 1.2 2015/06/17 10:51:08 jca Exp $

- Force building standalone library (e.g., put headers in appropriate
  place).
- workaround unavailable option (unused anyway)

--- lib/talloc/wscript.orig	Thu Jun  4 14:01:19 2015
+++ lib/talloc/wscript	Fri Jun  5 13:22:39 2015
@@ -40,6 +40,7 @@ def configure(conf):
     conf.RECURSE('lib/replace')
 
     conf.env.standalone_talloc = conf.IN_LAUNCH_DIR()
+    conf.env.standalone_talloc = True
 
     conf.env.disable_python = getattr(Options.options, 'disable_python', False)
 
@@ -52,8 +53,6 @@ def configure(conf):
             conf.define('USING_SYSTEM_PYTALLOC_UTIL', 1)
 
     conf.env.TALLOC_COMPAT1 = False
-    if conf.env.standalone_talloc:
-        conf.env.TALLOC_COMPAT1 = Options.options.TALLOC_COMPAT1
 
     conf.CHECK_XSLTPROC_MANPAGES()
 
